doc/cookbook.rst

```diff
@@ -26,31 +26,31 @@ between the calibration images and the science images.
 ```
 run ncpus 1
 run spectrograph lris_red
-out verbose 2
-out overwrite True
-out sorted lris_red_qpair
+output verbosity 2
+output overwrite True
+output sorted lris_red_qpair
 
 # Reduce
-arc calibrate id_pix 131.299,400.73,474.20,499.6,946.787,1423.99,2246.698,2723.903
-arc calibrate id_wave 5462.27,5771.21,5854.1101,5883.5252,6404.018,6967.35,7950.36,8523.78
-trace disp direction 0
-trace orders pcatilt 1,1,1
-trace orders tilts spca
-reduce bgsubtraction method bspline
-pixflat comb method median
-pixflat comb rej_level [10.0,10.0]
-pixflat norm recnorm False
-reduce usebias bias
+arc calibrate IDpixels 131.299,400.73,474.20,499.6,946.787,1423.99,2246.698,2723.903
+arc calibrate IDwaves 5462.27,5771.21,5854.1101,5883.5252,6404.018,6967.35,7950.36,8523.78
+trace dispersion direction 0
+trace slits tilts params [1,1,1]
+trace slits tilts method spca
```

Comment:
I presume the Development suite is going to barf.  But that's ok.
And I'm willing to make it work so that I understand all these changes.